src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitService.kt

```diff
@@ -350,6 +365,13 @@ class VisitService(
   fun cancelVisit(reference: String, cancelOutcome: OutcomeDto): VisitDto {
     val visitEntity = visitRepository.findBookedVisit(reference) ?: throw VisitNotFoundException("Visit $reference not found")
 
+    // check if the visit being cancelled is in the past
+    if (isBookingStartDateInPast(visitEntity))
+      throw ExpiredVisitAmendException(
+        "Visit with booking reference - ${visitEntity.reference} is in the past, it cannot be cancelled",
+        ExpiredVisitAmendException("trying to cancel an expired visit")
+      )
+
```

Comment:
I always put the {} with my if's your call but do like things to be consistant

if (isBookingStartDateInPast(visitEntity)) {
       throw ExpiredVisitAmendException(
        "Visit with booking reference - ${visitEntity.reference} is in the past, it cannot be cancelled",
        ExpiredVisitAmendException("trying to cancel an expired visit")
      )
}

Reply:
Agree, updated.